mllib/src/main/scala/org/apache/spark/ml/feature/LSH.scala

```diff
@@ -147,6 +148,15 @@ private[ml] abstract class LSHModel[T <: LSHModel[T]]
     modelSubsetWithDistCol.sort(distCol).limit(numNearestNeighbors)
   }
 
+  private[feature] def approxNearestNeighbors(
+      dataset: Dataset[_],
+      key: Vector,
+      numNearestNeighbors: Int,
+      singleProbe: Boolean,
+      distCol: String): Dataset[_] = {
+    approxNearestNeighbors(dataset, key, numNearestNeighbors, singleProbe, distCol, 0.05)
```

Comment:
Thanks!!!Your code help me a lot, and can I ask questions in Chinese(Maybe by sending you an email~)^_^